### DISCUSSION TITLE
Удаление фона вокруг выделенного поля в кадре видео

### DISCUSSION THREAD

**Discussion by jestemvision** [wladradchenko/wunjo.wladradchenko.ru#27] [General]
Владислав, приветствую!
Если я верно, понял, то Вы вели речь о том, что реализована возможность удалять фон вокруг выделенного участка (объекта) в кадре видео. Интересует не удаление объекта на фоне, а обратная задача (с сохранением прозрачности удаленных участков фона)

**Comment by wladradchenko** [comment]
Привет. Функционал уже реализован. Будет доступен в релизе 1.6. Сейчас его можно уже использовать из ветки v1.6, В [документации](https://github.com/wladradchenko/wunjo.wladradchenko.ru/wiki) информация как установить, однако я бы рекомендовал дождаться релиза, так как ветка 1.6 в разработке и внутри может что-то не работать или сломаться.

**Reply by jestemvision** [reply]
Верно-ли предполагаю, что на Маке не получится использовать?

**Reply by wladradchenko** [reply]
Конкретно эту функцию получится запустить на маке https://github.com/wladradchenko/wunjo.wladradchenko.ru/issues/14 . Но билд нужно будет собирать самостоятельно по документации для маков с новыми чипами. Однако модель для сегментации будет маленькая для CPU, она работает хуже в точности, чем большая модель для GPU

**Reply by jestemvision** [reply]
Значит, пролетаю со своим М1 на Mac-mini.

**Reply by wladradchenko** [reply]
> Значит, пролетаю со своим М1 на Mac-mini.

 Почему. Просто собирите билд по документации, там описаны все шаги.

**Reply by jestemvision** [reply]
Попробую, но у меня никогда и ничего из такого рода вещей не получается. Айтишники составляют инструкции для айтишников. Вроде бы всё делаешь по инструкции, но в конце всё равно не работает. У меня даже понятия нет, что на вашем проф. слэнге означает "билд" (!) :)
